### ISSUE TITLE
Reload happens before start of service 

### ISSUE THREAD

**Issue by jasonmcintosh** [johnbellone/consul-cookbook#407]
When testing, a reload of the service is occurring BEFORE consul has fully started (the delay happens almost immediately after the service start).  This seems to cause consul to fail to start causing some tests to fail.  I added a sleep block as a stop-gap:
```
ruby_block 'delay_reload_consul' do
  block do
    sleep 5
  end
end
```
After a start of the service, but there's probably a better/cleaner way to handle this.

**Comment by johnbellone** [comment]
What platform and version?

Sent from my iPhone
JB

**Comment by jasonmcintosh** [comment]
CentOS 7, x86-64.  COULD be a bug in consul, or in the service startup arguments.  Just noticed it in here, thought I'd report it.  I'll probably post a message to Consul as well.

**Comment by legal90** [comment]
It should be fixed by #427

**Comment by MichaelKueller** [comment]
actually this particular behaviour will be fixed with #433
